src/Tasks/XmlPoke.cs

```diff
@@ -223,7 +223,14 @@ public override bool Execute()
 
             if (iter.Count > 0)
             {
+#if RUNTIME_TYPE_NETCORE
```

Comment:
Yeah its weird because they're extension methods in .NET Core so I wasn't sure what the feature should be called?  So I just used `RUNTIME_TYPE_NETCORE` to ensure that older behavior is kept...